client/src/Templates/Layout/Grid.js

```diff
@@ -14,7 +14,8 @@ const Grid = styled.div`
     grid-template-columns: 220px 1fr;
   }
   @media only screen and (min-width: 1600px) {
-    grid-template-columns: 1fr 220px minmax(360px, 1160px) 220px 1fr;
+    // 230px is minimal value for the column width for the last sidebar menu item - is it acceptable for the main content window?
+    grid-template-columns: 1fr 230px minmax(360px, 1160px) 220px 1fr;
```

Comment:
Perhaps makes the sides the same size?

Reply:
That will be discussed with the team. See the question in the comment above.